apparmor.d/groups/gvfs/gvfsd-metadata

```diff
@@ -3,7 +3,6 @@
 # Copyright (C) 2022 Alexandre Pujol <alexandre@pujol.io>
 # SPDX-License-Identifier: GPL-2.0-only
 
-abi <abi/3.0>,
```

Comment:
Put it back.

Reply:
Intermediary commit. It's fixed in the latest.